kustomize/README.md

```diff
@@ -16,105 +16,123 @@ ngc_api_key=your_api_key
 EOF
 ```
-6. Transplant right service CNAME according to auto generated certificate
-```shell
-export CALLBACK_URL=https:\\/\\/$(echo agent-morpheus-client.$(oc projects | grep "*" | awk -F '* ' '{print $2}').svc:8443)
-find . | grep agent-morpheus-config.json | xargs -i sed -i "s/CALLBACK_URL_PLACEHOLDER/$CALLBACK_URL/g" {}
 
+6. Update `Agent Morpheus` configuration file with the correct callback URL for the client service.
 
+```shell
+export CALLBACK_URL="https://agent-morpheus-client.$(oc project -q).svc:8443"
+find . -type f -name 'agent-morpheus-config.json' -exec sed -i "s|CALLBACK_URL_PLACEHOLDER|$CALLBACK_URL|g" {} +
 ```
-7. Deploy agent-morpheus + agent-morpheus-client to your namespace. The default configuration will use local embeddings and remote NIM LLM
+
+7. Deploy `Agent Morpheus` and `Agent Morpheus Client` using default settings (local embeddings and remote NVIDIA NIM LLM)
+
 ```shell
 oc kustomize base |  oc apply -f - -n $YOUR_NAMESPACE_NAME
 ```
 
-8. Alternatively, if you want to deploy agent-morpheus with our self-hosted LLM or fully remote services, then run
+8. Alternatively, to deploy `Agent Morpheus` with a self-hosted LLM or fully remote service, run one of the following commands:
+
 ```shell
 # Deploy agent morpheus with local llama3.1 LLM 
 oc kustomize overlays/local-llama3.1-70b-4bit |  oc apply -f - -n $YOUR_NAMESPACE_NAME
 # Deploy agent morpheus with remote llama-3.1-70b-4bit LLM
 oc kustomize overlays/remote-nim-all |  oc apply -f - -n $YOUR_NAMESPACE_NAME
 ```
-9. If you didn't deploy yet self-hosted embedding model and one of the above LLMs, first fetch latest version of agent-morpheus git submodule
+
+9. If you have not yet deployed the self-hosted embedding model and one of the LLM options above, first fetch the latest version of the `Agent Morpheus` Git submodule.
+
 ```shell
-git submodule init --recursive
-git submodule update --merge --recursive
+git submodule update --init --recursive --merge
 # Or if the local submodule sha is not updated to latest one, you can fetch it from remote tracking branch
 git submodule update --remote --merge --recursive
 ```
-10. Follow the instructions below
-    1. from your local repository
-       ```shell
-       cd ../agent-morpheus-models    
-        ```
-       [open instructions locally](../agent-morpheus-models/README.md)
-    2. [Open instruction in browser](https://github.com/RHEcosystemAppEng/agent-morpheus-models)
 
-11. Create the OAuthClient with the same secret and using the generated route
+10. Follow the instructions to prepare the model
+    1. [Open instruction from local repository](../agent-morpheus-models/README.md)
+    2. [Open instruction in browser](https://github.com/RHEcosystemAppEng/agent-morpheus-models)
 
+11. Create the `OAuthClient` using the secret from step 5 and generated route
+export OAUTH_CLIENT_SECRET=some-long-secret-used-by-the-oauth-client
 ```bash
 oc apply -f - <<EOF
 apiVersion: oauth.openshift.io/v1
 kind: OAuthClient
```

Comment:
```suggestion
```bash
export OAUTH_CLIENT_SECRET=some-long-secret-used-by-the-oauth-client
oc apply -f - <<EOF
apiVersion: oauth.openshift.io/v1
kind: OAuthClient
```